README.md

```diff
@@ -0,0 +1,82 @@
+# Digital Calendar and Photoframe
+
+# Table of Contents
+
+- [1) Hardware](#1-hardware)
+   - [1.1) Hardware Requirements](#11-hardware-requirements)
+   - [1.2) Hardware Setup](#12-hardware-setup)
+- [2) Software](#2-software)
+   - [2.1) Install Dependencies](#21-install-dependencies)
+   - [2.2) Code Customization Options](#22-code-customization-options)
+   - [2.3) Running the Digital Calendar and Photo frame](#23-running-the-digital-calendar-and -photo-frame)
+
+# 1) Hardware
+
+## 1.1) Hardware Requirements
+
+- [Any 96Boards CE](https://www.96boards.org/products/ce/)
+- [HDMI + USB Touchscreen](https://www.arrow.com/en/products/96boards-display-7/linksprite-technologies-inc)
+
+## 1.2) Hardware Setup
+
+- Make sure the touchscreen is plugged in.
```

Comment:
Better say, "Make sure the touchscreen is connected to 96Boards CE using HDMI cable and USB"?

Reply:
ack